src/vue/common/EmailGetter.vue

```diff
@@ -104,9 +123,60 @@ export default {
         return ''
       }
     },
+
+    async copyToClipboard (data) {
+      try {
+        await navigator.clipboard.writeText(data)
```

Comment:
navigator.clipboard.writeText still under "draft" state and unsupported by most browsers. pls use something more stable, for example, you can use the approach from our ClipboardField component